src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -666,6 +806,72 @@ components:
         - photoLocation
         - validOnDate
 
+    GenerateAnonymousElectorDocumentRequest:
+      title: GenerateAnonymousElectorDocumentRequest
+      description: Request body to generate an Anonymous Elector Document (AED)
+      type: object
+      properties:
+        gssCode:
+          type: string
+          example: E12345678
+          minLength: 9
+          maxLength: 9
+          description: GSS code of the ERO responsible for the application
+        sourceType:
+          $ref: '#/components/schemas/SourceType'
+        sourceReference:
+          type: string
+          example: 1f0f76a9a66f438b9bb33070
+          description: Reference in the source application that this Temporary Certificate is for.
+        applicationReference:
+          type: string
+          example: V3JSZC4CRH
```

Comment:
I know its only an example, but might be worth checking with @matt-wills-valtech for the format of an application reference for an AED.
We know that Voter Card's start with a `V`, and I believe that other application types that originate in IER start with other letters to designate Postal, or Proxy or Overseas 
I know that AED's do not originate in IER, but I wonder if they have a different prefix?

Reply:
I've no idea, but Neil has already asked the BAs what the format of the electoral roll number is, so can he add this to the list! 😄

Reply:
Posed the question to the BAs but I don't think we will get any answers in time for this PR.